vtkext/private/module/F3DLog.cxx

```diff
@@ -68,3 +68,9 @@ void F3DLog::SetStandardStream(StandardStream mode)
       break;
   }
 }
+
+//----------------------------------------------------------------------------
+F3DLog::Severity F3DLog::GetVerboseLevel()
```

Comment:
this method is not needed, you can just use F3DLog::VerboseLevel where you needed it.